configs/prow-dev/config/plugins.yaml

```diff
@@ -79,6 +79,9 @@ external_plugins:
       events:
         - issue_comment
         - pull_request_review_comment
+        - pull_request-review
```

Comment:
pull_request-review?

Reply:
Ref: https://docs.github.com/en/free-pro-team@latest/developers/webhooks-and-events/webhook-events-and-payloads#pull_request_review

Reply:
Good catch!

Reply:
Fixed.